### PR TITLE
[ipcamera] Fix Reolink does not detect bad user or passwords.

### PR DESCRIPTION
**Pull request by Skinah**
Reolink cameras would not tell a user when the wrong user or password was entered, these changes make setup easier.

### CODE REVIEW

File: bundles/org.openhab.binding.ipcamera/src/main/java/org/openhab/binding/ipcamera/internal/onvif/OnvifCodec.java [hunk 1]
```diff
@@ -50,8 +50,28 @@ public void channelRead(@Nullable ChannelHandlerContext ctx, @Nullable Object ms
...
+                    case 401:
+                        if (!response.headers().isEmpty()) {
+                            String authenticate = "";
+                            for (CharSequence name : response.headers().names()) {
+                                for (CharSequence value : response.headers().getAll(name)) {
+                                    if ("WWW-Authenticate".equalsIgnoreCase(name.toString())) {
+                                        authenticate = value.toString();
+                                        logger.debug(
+                                                "ONVIF replied with WWW-Authenticate header:{}, camera may require ONVIF Profile-T support.",
+                                                authenticate);
```
Comment (lolodomo) [inline bundles/org.openhab.binding.ipcamera/src/main/java/org/openhab/binding/ipcamera/internal/onvif/OnvifCodec.java:65]: It is a detail but you could simply declare the variable inside the if as it is only used for the log:
String authenticate = value.toString();

Comment (lolodomo) [reply bundles/org.openhab.binding.ipcamera/src/main/java/org/openhab/binding/ipcamera/internal/onvif/OnvifCodec.java:65]: Or even better, not declare at all this variable ;)
And just use `value.toString()` as parameter of logger.debug

Comment (Skinah) [reply bundles/org.openhab.binding.ipcamera/src/main/java/org/openhab/binding/ipcamera/internal/onvif/OnvifCodec.java]: Fixed and thank you for reviewing it.

**Review by lolodomo — APPROVED** [review]
LGTM, thank you

**Pull request merged**